src/components/Modals/RulesModal.vue

```diff
@@ -28,7 +28,7 @@
 
                     <p style="text-align: center; font-size: 24px;"><b>Playing Cards</b></p>
                     <li><h5>Instruction</h5>
-                      <div>
+                      <div >
```

Comment:
It looks like all of these changes are just white space. Rever the file back to the original.

Reply:
fixed